.github/workflows/release.yml

```diff
@@ -26,20 +26,30 @@ jobs:
         with:
           # Allow goreleaser to access older tag information.
           fetch-depth: 0
+      
       - uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
           cache: true
+      
       - name: Import GPG key
         uses: crazy-max/ghaction-import-gpg@cb9bde2e2525e640591a934b1fd28eef1dcaf5e5 # v6.2.0
         id: import_gpg
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           passphrase: ${{ secrets.PASSPHRASE }}
+      
+      - name: Set version without 'v' prefix
+        id: version
+        run: |
+          VERSION=${GITHUB_REF_NAME#v}
```

Comment:
So that you saying is that you expect the Tag to be stored in GITHUB_REF_NAME? According to docs that could be short branch name or tag, not clear when is what? Did you tested it?

Reply:
it should be a tag because this workflow only runs on tags.  Can't test outside the context of this workflow so we need run it on our next release and validate.